railties/lib/rails/generators/rails/plugin/templates/test/test_helper.rb

```diff
@@ -20,7 +20,7 @@
 
 # Load fixtures from the engine
 if ActiveSupport::TestCase.respond_to?(:fixture_path=)
-  ActiveSupport::TestCase.fixture_path = File.expand_path("../fixtures", __FILE__)
+  ActiveSupport::TestCase.fixture_path = File.expand_path("../dummy/test/fixtures", __FILE__)
```

Comment:
It should use `options[:dummy_path]` and I think it would need to go up another folder. `"../../<%= options[:dummy_path] -%>/test/fixtures"`